lib/moon/components/drawer.ex

```diff
@@ -7,6 +7,7 @@ defmodule Moon.Components.Drawer.Dialog do
   alias Moon.Icons.ControlsClose
 
   prop(close, :event)
+  prop(has_no_divider, :boolean, default: false)
```

Comment:
I think it should be positive, not double negative.

So has_divider and default true.